src/utils/expenses.js

```diff
@@ -0,0 +1,59 @@
+const { Op } = require('sequelize');
+
+const { getValidString } = require('./validation');
+const { getErrorWithStatus } = require('./getError');
+
+const isExpenseValid = (expense) => {
+  const { title, category, spentAt, note } = expense;
+
+  const textFields = {
+    title,
+    category,
+    spentAt,
+    note,
+  };
+
+  Object.entries(textFields).forEach(([key, value]) => {
+    const field = key.toString();
+
+    if (field === 'note' && !field) {
```

Comment:
The condition `if (field === 'note' && !field)` seems incorrect. It should likely check if `value` is not provided for the 'note' field, i.e., `if (field === 'note' && !value)`.